src/Blocks/components/card-inline/card-inline-admin.scss

```diff
@@ -119,6 +121,35 @@
 			@include media(desktop up) {
 				grid-template-columns: repeat(3, 1fr);
 			}
+
+			&--break-words {
+				li {
+					text-overflow: ellipsis;
+					overflow: auto;
+					transition: {
+						duration: 0.3s;
+						property: color;
+						timing-function: ease-in-out;
+					}
+
+					&:hover {
+						color: var(--global-colors-esf-gray-950);
+
+						span {
+							white-space: inherit;
+						}
+					}
+
+					strong {
+						display: block;
+					}
+
+					span {
+						white-space: nowrap;
+						padding-left: 0.9375rem;
```

Comment:
Can logical CSS properties be used here?

Reply:
this will e refactored in the future so no need for this at this point